test/test.cpp

```diff
@@ -247,6 +247,49 @@ static void testParseErrorHandling() {
     } catch (const std::runtime_error& err) {
         assert(std::string(err.what()).find("Invalid") != std::string::npos);
     }
+
+    // test invalid polygon
+    try {
+        readGeoJSON("test/fixtures/invalid-polygon.json", false);
+        assert(false && "Should have thrown an error");
+    } catch (const std::runtime_error& err) {
+        assert(std::string(err.what()).find("described by 4") != std::string::npos);
+    }
+
+    try {
+        readGeoJSON("test/fixtures/invalid-polygon-2.json", false);
+        assert(false && "Should have thrown an error");
+    } catch (const std::runtime_error& err) {
+        assert(std::string(err.what()).find("array") != std::string::npos);
```

Comment:
Why not just check the whole message?

Reply:
This was the case with existing test cases, so I tried to keep it the same. Should it be changed?